### DISCUSSION TITLE
Binding to cmd key on mac don't seem to work

### DISCUSSION THREAD

**Discussion by kristoferlund** [ghostty-org/ghostty#3497] [Q&A] [answered]
Does the super/cmd/command key have to be enabled somehow before being used or is this a bug? 

The following escape sequence works well with the `ctrl` key but not with the `cmd` key.

```
keybind = ctrl+.=text:\x0D
keybind = cmd+.=text:\x0D
```

**Comment by jwadolowski** [comment]
I've just tried `cmd+.` and indeed it seems to be ignored.

```
# does NOT work
keybind = cmd+.=esc:.

# both work
keybind = cmd+;=esc:.
keybind = alt+.=esc:.
```

```console
$ ghostty +list-keybinds | rg 'esc:\.'
keybind = alt+period=esc:.
keybind = super+semicolon=esc:.
keybind = super+period=esc:.
```

Here's Ghostty inspector output:

![cleanshot_2024-12-28T005955_qaRUVZ3B@2x](https://github.com/user-attachments/assets/1a512fa9-c632-464b-a2d2-a7d8415425c1)

**Comment by jwadolowski** [comment]
Related discussions: 
- https://github.com/ghostty-org/ghostty/discussions/2881
- https://github.com/ghostty-org/ghostty/discussions/3858

FYI, I've just upgraded to [1.0.1](https://ghostty.org/docs/install/release-notes/1-0-1), but `keybind = cmd+.=esc:.` is still ignored by Ghostty.

**Comment by jarodtaylor** [comment]
Honestly, for a 'native Mac' application, there are some weird "non-mac" things with the key bindings. I wanted to drop Warp/Kitty for this, but I'll have to wait until it matures or we get better documentation on achieving some things. 

CMD + A should select the input text, not the entire terminal. The lack of mouse use (when you need it in the input also makes it hard to move from Warp/Kitty), and the shift + click is very laggy. I don't need 80% of what Warp offers, but its ergonomics make it hard to move away from.

**Comment by jwadolowski** [comment, marked as answer]
The issue has been fixed in [1.1.0](https://ghostty.org/docs/install/release-notes/1-1-0). `keybind = cmd+period=esc:.` works correctly now.

**Reply by 00-kat** [reply]
I can't find out which one fixed it, I think it was #4591.

**Reply by 00-kat** [reply]
Found it, it was definitely #4591.  See: #2881 #4540.
